vtm/src/org/oscim/layers/tile/bitmap/BitmapTileLayer.java

```diff
@@ -47,13 +49,28 @@ public FadeStep(int zoomStart, int zoomEnd, float alphaStart, float alphaEnd) {
             this.alphaStart = alphaStart;
             this.alphaEnd = alphaEnd;
         }
+
+        public FadeStep(double scaleStart, double scaleEnd, float alphaStart, float alphaEnd) {
+            this.scaleStart = scaleStart;
+            this.scaleEnd = scaleEnd;
+            this.alphaStart = alphaStart;
+            this.alphaEnd = alphaEnd;
+        }
     }
 
     public BitmapTileLayer(Map map, TileSource tileSource) {
         this(map, tileSource, CACHE_LIMIT);
     }
 
+    public BitmapTileLayer(Map map, TileSource tileSource, float transparency) {
+        this(map, tileSource, CACHE_LIMIT, transparency);
+    }
+
     public BitmapTileLayer(Map map, TileSource tileSource, int cacheLimit) {
+        this(map, tileSource, cacheLimit, 0f);
+    }
+
+    public BitmapTileLayer(Map map, TileSource tileSource, int cacheLimit, float transparency) {
```

Comment:
More user intuitive to have the `alpha` as parameter (common from colors), instead of the `transparency`?

Reply:
I always hate alpha because it too technical and is confusing (as alpha 0 is in fact full transparency) but I can change it if you wish.

Reply:
Yes please let's use alpha as is what most are used to.